### ISSUE TITLE
updates-testing failed in TestKdumpNFSAnsible.testBasic

### ISSUE THREAD

**Issue by cockpituous** [cockpit-project/cockpit#20779]
The job `fedora-40/updates-testing` failed on commit 762bf1eb5adfcab704fd03e0f569fd59b288bc84.

Log: https://cockpit-logs.us-east-1.linodeobjects.com/pull-0-762bf1eb-20240720-013002-fedora-40-updates-testing/log.html

**Comment by martinpitt** [comment]
This already happened in #20786, so this is real. Urgent, let's block the bodhi  update.

**Comment by martinpitt** [comment]
I ran it three times locally, and it always passes. This smells more like a flake, perhaps due to running low on RAM, or too many parallel VMs?

**Comment by martinpitt** [comment]
Ah wait -- in the other bug it happened in /daily, not /updates-testing. The [NFS machine](https://cockpit-logs.us-east-1.linodeobjects.com/pull-0-762bf1eb-20240720-013002-fedora-40-updates-testing/TestKdumpNFSAnsible-testBasic-fedora-40-127.0.0.2-2403-FAIL.log.gz) shows no failure at all, neither does the kdump machine log. But that doesn't show the *interesting* bit (the console log), which is the really frustrating bit -- we added this *precisely* to debug such issues. But we print the *wrong* machine.

The [weather report](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20769-79d45c69-20240717-054115-fedora-40-devel/log.html#48) has a 10% failure rate of this as well, but it's different. Curiously I just got the same failure on my fourth local run.

I sent #20787 this should help a bit with debugging and that .multipleTargets crash flake.
